src/bundle/Controller/SuggestionController.php

```diff
@@ -17,14 +18,23 @@ final class SuggestionController extends AbstractController
 {
     private SuggestionService $suggestionService;
 
+    private ConfigResolverInterface $configResolver;
+
     public function __construct(
-        SuggestionService $suggestionService
+        SuggestionService $suggestionService,
+        ConfigResolverInterface $configResolver
     ) {
         $this->suggestionService = $suggestionService;
+        $this->configResolver = $configResolver;
     }
 
     public function suggestAction(SuggestionQuery $suggestionQuery): JsonResponse
     {
+        $minQueryLength = $this->configResolver->getParameter('search.suggestion.min_query_length');
+        if (mb_strlen($suggestionQuery->getQuery()) < $minQueryLength) {
```

Comment:
just to be sure, `$suggestionQuery->getQuery()) < $minQueryLength` or `$suggestionQuery->getQuery()) <= $minQueryLength`?

Reply:
It's fine, the condition is reversed - quit if query length is lower than configured limit.